lfp_online/BinFileReaderProcessor.cpp

```diff
@@ -164,7 +164,8 @@ std::string BinFileReaderProcessor::binFileDuration(std::string file_path) {
 		return binFileDurationFromNSampes(fsize / 2 / buffer->CHANNEL_NUM);
 	}
 
-	return 0;
+    // return 0; --  bug?: int -> null pointer -> string (undefined behaviour)
```

Comment:
@igridchyn The intention was to return string "0"? Am I right?

Reply:
It should be, yes, since a string should be returned.